x-pack/plugins/security_solution/public/common/components/markdown_editor/plugins/osquery/plugin.tsx

```diff
@@ -156,19 +156,28 @@ const OsqueryEditorComponent = ({
 
 const OsqueryEditor = React.memo(OsqueryEditorComponent);
 
-export const plugin = {
-  name: 'osquery',
-  button: {
-    label: 'Osquery',
-    iconType: 'logoOsquery',
-  },
-  helpText: (
-    <div>
-      <EuiCodeBlock language="md" fontSize="l" paddingSize="s" isCopyable>
-        {'!{osquery{options}}'}
-      </EuiCodeBlock>
-      <EuiSpacer size="s" />
-    </div>
-  ),
-  editor: OsqueryEditor,
+export const plugin = ({
+  interactionsUpsellingMessage,
+}: {
+  interactionsUpsellingMessage: string | null;
+}) => {
+  return {
+    name: 'osquery',
+    button: {
+      label: interactionsUpsellingMessage ?? 'Osquery',
+      'aria-label': 'aria-label-test1',
+      iconType: 'logoOsquery',
+      isDisabled: !!interactionsUpsellingMessage,
+      'data-test-subj': 'test-test-test',
```

Comment:
rename or maybe remove since it appears unused?

Reply:
oh man, thanks for catching this.. was playing around with plugin and forgot to remove this line